apps/authentication/mfa/email.py

```diff
@@ -6,20 +6,25 @@
 from common.utils.verify_code import SendAndVerifyCodeUtil
 from settings.utils import get_login_title
 from .base import BaseMFA
+from ..const import MFAType
 
 email_failed_msg = _("Email verify code invalid")
 
 
 class MFAEmail(BaseMFA):
-    name = 'email'
-    display_name = _('Email')
+    name = MFAType.Email.value
+    display_name = MFAType.Email.name
     placeholder = _('Email verification code')
 
     def _check_code(self, code):
         assert self.is_authenticated()
         sender_util = SendAndVerifyCodeUtil(self.user.email, backend=self.name)
-        ok = sender_util.verify(code)
-        msg = '' if ok else email_failed_msg
+        ok = False
+        msg = ''
+        try:
+            ok = sender_util.verify(code)
+        except Exception as e:
+            msg = str(e)
         return ok, msg
 
     def is_active(self):
```

Comment:
There don't seem to be any immediate issues with this Python code snippet. However, some general advice could include:

1. Make sure all imports inside functions have the latest version of their respective libraries (common, settings).
2. Add type hints for better readability and maintainability.
3. Use more descriptive variable names (`verify` instead of `ok`) that make it clear what you're checking.
4. Handle exceptions properly in error messages.

The current structure doesn't look suspicious but might benefit from being reviewed further before going live.